cluster/gce/util.sh

```diff
@@ -30,13 +30,30 @@ else
   exit 1
 fi
 
+function get_latest_gci_image() {
+  # GCI milestone to use
+  milestone="53"
+
+  # First try to find an actie (non-deprecated) image on this milestone.
+  gci_images=( $(gcloud compute images list --project google-containers \
+      --no-standard-images --sort-by="~creationTimestamp" \
+      --regexp="gci-[a-z]+-${milestone}-.*" --format="table[no-heading](name)") )
+
+  # If no active image is available, search across all deprecated images.
+  if [[ ${#gci_images[@]} == 0 ]] ; then
+    gci_images=( $(gcloud compute images list --project google-containers \
+        --no-standard-images --show-deprecated --sort-by="~creationTimestamp" \
+        --regexp="gci-[a-z]+-${milestone}-.*" --format="table[no-heading](name)") )
+  fi
+
+  echo "${gci_images[0]}"
```

Comment:
Humm.. I tried return "${gci_images[0]}" (and also without quotes), but I get error "numeric argument required". I couldn't find any other way to return a string from a bash function (but I am not a bash expert).